.github/workflows/android-monkey.yaml

```diff
@@ -50,7 +50,14 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v4
         with:
-          fetch-depth: 100 # enough to get all commits for the current day
+          fetch-depth: 1
```

Comment:
Current version is calculated based on commit history.

Reply:
Which doesn't matter here at all. Anyway.